tests/testProcessCcd.py

```diff
@@ -133,7 +133,13 @@ def testProcessCcd(self):
                     print("image mean = %r, stdDev = %r" % (imMean, imStdDev))
                     print("variance mean = %r, stdDev = %r" % (varMean, varStdDev))
                     print("psf Ixx = %r, Iyy = %r, Ixy = %r" % (psfIxx, psfIyy, psfIxy))
-
+                    
+                    # TODO: these thresholds are larger than they ought to be because the 
+                    # background modeling differs in py2 and py3.
+                    # The values asserted here are the precise result that py2 gives. The precision below 
+                    # should be tightened and this comment removed in the course of DM-8017.
+                    # Also see: https://community.lsst.org/t/difference-in-py2-and-py3-background-models/1240
+                    
```

Comment:
I guess disregard the past comment as I got to this last. As there is a ticket to do this work, I see no need to do the work on this ticket.